<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>m2PreferencePage</class>
 <widget class="QWidget" name="m2PreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>External Programs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Registration preferences</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Registration and 3D reconstruction requires external programs to work.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="transformixLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="elastixLabel">
       <property name="text">
        <string>elastix:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="transformixLabel">
       <property name="text">
        <string>transformix:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="elastixButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="elastixLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="transformixButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ImzML import preferences</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Overview spectrum binning for processed centroid spectrum data. This value is used to generate overview spectra only.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,4">
     <item row="0" column="0">
      <widget class="QLabel" name="elastixLabel_2">
       <property name="text">
        <string>Number of bins:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spnBxBins">
       <property name="minimum">
        <number>2000</number>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Overview Spectrum view preferences</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useMaxIntensity">
     <property name="text">
      <string>Enable use max intensity in x range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useMinIntensity">
     <property name="text">
      <string>Enable use min intensity in x range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Load settings:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxMaskImage">
     <property name="toolTip">
      <string>A new image node is added to the DataManager. 
Maskimage: background (pixel value = 0) and all valid spectra/pixels (pixel value = 1) .</string>
     </property>
     <property name="text">
      <string>Show mask image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxIndexImage">
     <property name="toolTip">
      <string>A new image node is added to the DataManager. 
Index-image: background (pixel value = 0) and for all valid spectra/pixels the associated spectrum-index (pixel values = {1,...,n}) is given.</string>
     </property>
     <property name="text">
      <string>Show index image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkBxNormalizationImage">
     <property name="toolTip">
      <string>A new image node is added to the DataManager. 
Normalization-image: background (pixel value = 0) and for all valid spectra/pixels the associated spectrum-normalization factor (real pixel values) is given. E.g. if TIC normalization was selected for data loading, the Normalization-image containse the TIC factors for each spectrum/pixel.</string>
     </property>
     <property name="text">
      <string>Show normalization image</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>248</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
